src/app/about/page.tsx

```diff
@@ -0,0 +1,126 @@
+import CategoriesCard from "@/components/categories/categories-card";
+import CollectionCard from "@/components/collections/collections-card";
+import CreatorCard from "@/components/creators/creator-card";
+import Image from "next/image";
+
+const About = () => {
+    return (
+        <main className="flex min-h-screen flex-col items-center justify-between p-24">
+            <div className="max-w-6xl">
+                <div className="flex items-center justify-between gap-10 w-full">
+                    <div className=" flex items-center flex-col max-w-lg gap-5">
+                        <h2 className="font-sans text-6xl font-semibold ">
```

Comment:
I see a lot of `"font-sans"` usage, but there's global styles applied with this font. I think you can remove this `className`